pkg/cluster/cluster_info_scraper.go

```diff
@@ -40,6 +40,34 @@ func NewClusterInfoScraper(kubeConfig *restclient.Config) (*ClusterScraper, erro
 	}, nil
 }
 
+func (s *ClusterScraper) GetNamespaces() ([]*api.Namespace, error){
+	namespaceList, err := s.Core().Namespaces().List(metav1.ListOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("failed to list all namespaces in the cluster: %s", err)
+	}
+	fmt.Printf("There are %d namespaces\n", len(namespaceList.Items))
```

Comment:
don't forget to remove the fmt.Printf or change them to glog.V().Infof() before merging.